content/hi/docs/home/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: कुबेरनेट्स प्रलेखन  
+noedit: true
+cid: docsHome
+layout: docsportal_home
+class: gridPage gridPageHome
+linkTitle: "होम"
+main_menu: true
+weight: 10
+hide_feedback: true
+menu:
+  main:
+    title: "प्रलेखन"
+    weight: 20
+    post: >
+      <p>वैचारिक, ट्यूटोरियल और संदर्भ प्रलेखन  के साथ कुबेरनेट्स का उपयोग करना सीखें| आप <a href="/editdocs/" data-auto-burger-exclude data-proofer-ignore>डॉक्स में योगदान करने में</a>! भी मदद कर सकते हैं</p>
+description: >
+  कुबेरनेट्स कंटेनरीकृत अनुप्रयोगों के परिनियोजन, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेशन इंजन है| यह ओपन सोर्स प्रोजेक्ट को Cloud Native Computing Foundation द्वारा होस्ट किया गया है।
+overview: >
+  कुबेरनेट्स कंटेनरीकृत अनुप्रयोगों के परिनियोजन, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेशन इंजन है। यह ओपन सोर्स प्रोजेक्ट को Cloud Native Computing Foundation द्वारा होस्ट किया गया है|(<a href="https://www.cncf.io/about">CNCF</a>).
+cards:
+- name: concepts
+  title: "मूल बातें समझें"
+  description: "कुबेरनेट्स और इसकी मूलभूत अवधारणाओं के बारे में जानें|"
+  button: "अवधारणाओं को जानें"
+  button_path: "/hi/docs/concepts"
+- name: tutorials
+  title: "कुबेरनेट्स को उपयोग कर ने का प्रयास करें"
+  description: "कुबेरनेट्स में एप्लिकेशन को परिनियोजित करने का तरीका जानने के लिए ट्यूटोरियल का अनुसरण करें|"
```

Comment:
```suggestion
  description: "कुबेरनेट्स में एप्लिकेशन को डिप्लॉय करने के लिए ट्यूटोरियल का अनुसरण करें|"
```

Reply:
i think till then, we had not decided a global word for this!! so, thats why wasn't able to make changes
Thanks